doc/xbgpu.design.x.rst

```diff
@@ -91,7 +90,7 @@ thread grid, and could just as easily be dynamic.
 :c:macro:`NR_STATIONS_PER_TCM_X` and :c:macro:`NR_STATIONS_PER_TCM_Y` are the
 number of (dual-pol) antennas per warp matrix multiply. Keeping in mind that
 the "Y" station corresponds to rows (and to :c:var:`aSamples` temporary
-storage, with "X" corresponding to :c:var:`bSamples`), this is 8×4 (4×2 for
+storage, with "X" corresponding to :c:var:`bSamples`), this is 8×4 (8×2 for
```

Comment:
I've changed all instances of station to receiver. Unfortunately there will be some mixing of receiver versus antennas because TCC and the rest of katgpucbf use these different names. I've tried to use "receiver" when documenting the original TCC internals, and "antenna" when talking more generally about katgpucbf. Let me know if there are still places where things seems out of place.